MetaMask/metamask-extension
# Some icon fixes

## Explanation

Currently, there is a broken svg and proptype in the Icon component

This PR fixes the svg and uses the correct propType

## Screenshots/Screencaps

### Before
<img width="1432" alt="Screen Shot 2022-09-30 at 12 30 42 PM" src="https://user-images.githubusercontent.com/8112138/193343417-217eabe8-23c6-4406-9715-5447b5eb95e0.png">

### After
<img width="1440" alt="Screen Shot 2022-09-30 at 12 31 25 PM" src="https://user-images.githubusercontent.com/8112138/193343435-159c0bdf-d522-408c-ac8e-cd1f7a961001.png">

## Manual Testing Steps

- Go to the latest build of storybook in this PR
- Search "Icon" in storybook (should be in `component-library/icon`)
- Search "search" in the `Name` story

## Pre-Merge Checklist


### + If there are functional changes:
